cpp/tests/interop/to_arrow_device_test.cpp

```diff
@@ -711,6 +711,82 @@ TEST_F(ToArrowDeviceTest, StructColumn)
 template <typename T>
 using fp_wrapper = cudf::test::fixed_point_column_wrapper<T>;
 
+TEST_F(ToArrowDeviceTest, FixedPoint32Table)
+{
+  using namespace numeric;
+
+  for (auto const scale : {3, 2, 1, 0, -1, -2, -3}) {
```

Comment:
I'm not following this test fully, but we should make sure to use sufficiently large numbers so that the fp scale does not lead to a column full of zeroes.